src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -151,6 +151,7 @@
 
 (defmethod connection-with-timezone :sql-jdbc
   [driver database ^String timezone-id]
+  (sql-jdbc.conn/invalidate-pool-if-ssh-tunnel-closed! database)
```

Comment:
Unclear why this isn't covered...

Reply:
OK, thanks to the new `test-ssh-tunnel-reconnection-h2` test, this is now covered even for H2, and hence for the coverage check.

Reply:
This is finally covered, after a series of changes.